src/components/post/detail/index.tsx

```diff
@@ -72,35 +77,42 @@ function PostDetail() {
           </Icon>
 
           <Icon>
-            <Battle size={1.5} color='white' battleAbility={true} onClick={navigatePostBattle} />
+            {postDetail?.isBattlePossible ? (
+              <Battle size={1.5} color='white' battleAbility={true} onClick={navigatePostBattle} />
+            ) : (
+              <Battle size={1.5} color='white' battleAbility={false} />
+            )}
```

Comment:
이거 그냥 이렇게 해주시면 돼요 navigatePostBattle가 undefined면 prop 안 넣은 거랑 똑같으니까요~

```suggestion
            <Battle size={1.5} color='white' battleAbility={true} onClick={navigatePostBattle} />
```

Reply:
대결 불가 추천 글에 있어서는 대결 불가 아이콘을 render 해야 된다고 생각합니다. 제안해주신 소스코드는 대결 불가 추천 글도 대결 가능 아이콘이 뜨는 문제가 발생합니다~